ytsync/setup.go

```diff
@@ -37,16 +41,22 @@ func (s *Sync) walletSetup() error {
 		}
 	}
 	log.Debugf("Source channel has %d videos", numOnSource)
+	if numOnSource == 0 {
+		return nil
+	}
 
 	numPublished, err := s.daemon.NumClaimsInChannel(s.LbryChannelName)
 	if err != nil {
 		return err
 	}
 	log.Debugf("We already published %d videos", numPublished)
 
-	minBalance := (float64(numOnSource)-float64(numPublished))*publishAmount + channelClaimAmount
+	minBalance := (float64(numOnSource)-float64(numPublished))*(publishAmount+0.1) + channelClaimAmount
+	if numPublished > numOnSource {
+		util.SendErrorToSlack("something is going on as we published more videos than those available on source: %d/%d", numPublished, numOnSource)
```

Comment:
yes if we'll allow publishing to channels while we also keep them in sync then this will be the case. otherwise this happens when channel owners delete videos from their youtube channel